custom_model_runner/datarobot_drum/resource/deployment_config_helpers.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+from datarobot_drum.drum.exceptions import DrumCommonException
+from datarobot_drum.drum.common import TargetType
+
+
+def parse_validate_deployment_config_file(filename):
+    if filename is None:
+        return None
+    with open(filename) as f:
```

Comment:
What if the `filename` does not exists? Maybe add `try ... except`

Reply:
filename existence is checked in the arg parse validator